netket/hilbert/abstract_continuous_space_particle.py

```diff
@@ -3,56 +3,33 @@
 from .abstract_hilbert import AbstractHilbert
 
 
-class AbstractParticle(AbstractHilbert):
+class ContinuousHilbert(AbstractHilbert):
     """Abstract class for the Hilbert space of particles
     in continuous space.
     This class defines the common interface that
     can be used to interact with particles defined
     in continuous space.
     """
 
-    def __init__(
-        self, N: int, L: Tuple[float, ...], pbc: Union[bool, Tuple[bool, ...]]
-    ):
+    def __init__(self, domain: Tuple[float, ...], pbc: Union[bool, Tuple[bool, ...]]):
         """
         Constructs new ``Particles`` given specifications
          of the continuous space they are defined in.
 
         Args:
-            N: Number of particles
-            L: spatial extension in each spatial dimension
-            pbc: Whether or not to use periodic boundary
-                conditions for each spatial dimension.
-                If bool, its value will be used for all spatial
-                dimensions.
+            domain: range of the continuous quantum numbers
         """
-        self._N = N
-        self._L = L
-
-        if isinstance(pbc, bool):
-            pbc = [pbc] * len(self._L)
-
-        self._pbc = pbc
+        self._extent = domain
         if not len(self._L) == len(self._pbc):
             raise ValueError(
                 """`pbc` must be either a bool or a tuple indicating the periodicity of each spatial dimension."""
             )
-
         super().__init__()
 
     @property
-    def size(self) -> int:
-        return self._N * len(self._L)
-
-    @property
-    def n_particles(self) -> int:
-        r"""The number of particles"""
-        return self._N
-
-    @property
-    def extend(self) -> Tuple[float, ...]:
+    def extent(self) -> Tuple[float, ...]:
         r"""Spatial extension in each spatial dimension"""
-        return self._L
+        return self._extent
 
     @property
```

Comment:
Is PBC a property for hilbert spaces?

I guess what it tells us is basically if the wavefunctions in this space are supposed to be periodic or should decay to zero...

So it kind-of look like a property of the Hilbert space.

We agree on that?